README.md

```diff
@@ -4,22 +4,22 @@ Generates stream metadata for Fedora CoreOS using release metadata and overrides
 # Running
 Make sure you have latest go installed and then download the pacakge
 ```
-$ go get -u github.com/sinnykumari/fedora-coreos-stream-generator
+$ go get -u github.com/coreos/fedora-coreos-stream-generator
 ```
 Add the binary in your PATH variable. If you haven't done anything fancy, adding standard binary path where Go keeps binaries from package will be enough
 ```
 $ export PATH=$PATH:~/go/bin/
-$ fedora-coreos-stream-generator -release-index=<release index location for the stream>
+$ fedora-coreos-stream-generator -releases=<release index location for the stream> -output-file=stream.json
 ```
-Partial stream override can be given using option -override=<override json file location for the stream> . override.json need to be available locally to the system. https://github.com/coreos/fedora-coreos-streams repo will contains override for FCOS streams
+Partial stream override can be given using option -override=< /path/to/override.json> . override.json need to be available locally to the system. https://github.com/coreos/fedora-coreos-streams repo will contains override for FCOS streams
```

Comment:
Minor: looks like a stray space before `/path`? Also missing trailing periods a bit everywhere.

Reply:
Fixed! Looks like I don't have habit of adding periods in the last sentence of a paragraph. <-- such as this one xD